### ISSUE TITLE
Conversion of type 'AstroConfig' to type 'Required<AstroUserConfig>' - Error

### ISSUE THREAD

**Issue by felix-berlin** [shishkin/astro-pagefind#29]
I just updated to Astro v2.9.0, when I try to build my app with the following command: `astro check && tsc --noEmit && vue-tsc --noEmit && astro build` I get this error that prevents the build progress.

```bash
node_modules/astro-pagefind/src/pagefind.ts:14:19 - error TS2352: Conversion of type 'AstroConfig' to type 'Required<AstroUserConfig>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'AstroConfig' is missing the following properties from type 'Required<AstroUserConfig>': renderers, projectRoot, src, pages, and 6 more.

14         const c = config as Config;
                     ~~~~~~~~~~~~~~~~


Found 1 error in node_modules/astro-pagefind/src/pagefind.ts:14

```

**Comment by shishkin** [comment]
Sorry about that. TypeScript with ESM is still quite arcane magic to me.

It should be fixed in f31b3e1d99b4f81c423d23f5dc0fe3c43e32b1ea.

Please try [v1.2.3](https://github.com/shishkin/astro-pagefind/releases/tag/v1.2.3).

**Comment by felix-berlin** [comment]
Thank you for the quick response. The error is gone 😊
